src/Verify.MSTest/TestExecutionContext.cs

```diff
@@ -0,0 +1,20 @@
+namespace VerifyMSTest;
+
+public record TestExecutionContext(TestContext TestContext, Type TestClass)
+{
+    public Assembly TestAssembly { get; } = TestClass.Assembly;
```

Comment:
lets go with `Aeembly` and `Method as property names. the `Test` part is assumed